lib/log/json_formatter.rb

```diff
@@ -0,0 +1,26 @@
+require 'json'
+
+module Log
+  class JSONFormatter
+
+    def call(severity, time, progname, msg)
+      log_item = {
```

Comment:
Everything that is a cross cutting concern, meaning that it needs to be
done for application should NOT be done by the application. Shared library
is an OK way to do this, but best would be if done by logspout.

On Wed, Nov 11, 2015 at 4:06 PM Peter Marton <notifications@github.com>
wrote:

> In lib/log/json_formatter.rb
> <https://github.com/sspinc/csv2avro/pull/30#discussion_r44541463>:
>
> > @@ -0,0 +1,26 @@
> > +require 'json'
> > +
> > +module Log
> > +  class JSONFormatter
> > +
> > +    def call(severity, time, progname, msg)
> > +      log_item = {
>
> I didn't, because it's added by logspout anyways. I agree though, that it
> would probably make more sense to add the timestamp here and not there.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sspinc/csv2avro/pull/30/files#r44541463>.
>